documentation/data-protection.md

```diff
@@ -6,8 +6,19 @@ The [Data Protection Act](https://www.gov.uk/data-protection/the-data-protection
 
 ## What does data protection have to do with open data?
 
+In general, [open data](https://theodi.org/what-is-open-data) should not contain personal or sensitive personal data that could allow a living person to be identified. Data published to the [360Giving standard](http://www.threesixtygiving.org/standard/) should be anonymised to protect privacy as outlined in the ICO's [anonymisation code of practice](https://ico.org.uk/for-organisations/guide-to-data-protection/anonymisation/). There are cases where publishing personal data is in the public interest or where data can be published with the consent of the individual. We explore those cases below.
 
-[Open data](https://theodi.org/what-is-open-data) should not contain personal or sensitive personal data that could allow a living person to be identified (there are [exceptions](https://www.gov.uk/government/publications/dft-senior-officials-business-expenses-and-meetings-april-to-june-2014) but these don't apply to [360Giving](http://www.threesixtygiving.org/)). Data published to the [360Giving standard](http://www.threesixtygiving.org/standard/) should be anonymised to protect privacy as outlined in the ICO's [anonymisation code of practice](https://ico.org.uk/for-organisations/guide-to-data-protection/anonymisation/).
+## Publishing personal data with 360Giving
+We encourage publishers to carefully consider the value of sharing any personal data as part of their 360Giving publication and discourage sharing of sensitive personal data. 
```

Comment:
"We ... discourage sharing of sensitive personal data." - except we include fields that expect personal data in the standard?. I'd suggest something along the lines of:
"We encourage publishers to carefully consider the value of sharing any personal data as part of their 360Giving publication and exercise care when populating fields in the data standard that may expose sensitive personal data. "